api/post.go

```diff
@@ -603,6 +602,40 @@ func getPosts(c *Context, w http.ResponseWriter, r *http.Request) {
 
 }
 
+func getPostsSince(c *Context, w http.ResponseWriter, r *http.Request) {
+	params := mux.Vars(r)
+
+	id := params["id"]
+	if len(id) != 26 {
+		c.SetInvalidParam("getPostsSince", "channelId")
+		return
+	}
+
+	time, err := strconv.ParseInt(params["time"], 10, 64)
+	if err != nil {
+		c.SetInvalidParam("getPostsSince", "time")
+		return
+	}
+
+	cchan := Srv.Store.Channel().CheckPermissionsTo(c.Session.TeamId, id, c.Session.UserId)
+
+	if !c.HasPermissionsToChannel(cchan, "getPostsSince") {
+		return
+	}
+
+	pchan := Srv.Store.Post().GetPostsSince(id, time)
```

Comment:
do we want to set an upper limit in the query?  1000 items?  or something.  What happens when you've been gone for weeks and go into a busy channel.